extract/extract/extractors/otto_de.py

```diff
@@ -319,6 +334,12 @@ def _get_sustainability(
         sustainable_soup = BeautifulSoup(sustainability_information_html, "html.parser")
         labels.update(_get_sustainability_info(sustainable_soup))
 
+    # Due to the `filter` being empty, we expect that in `sustainability_information_htmls` there
+    # wouldn't be anything found for the non-sustainable products, so we set a default label
+    # UNAVAILABLE, so the non-sustainable products can be distinguished from the other products.
+    if not labels:
+        labels = {UNAVAILABLE: ""}
```

Comment:
If no sustainability strings are found, we could also directly return the `CertificateType.UNAVAILABLE` here. And use the other return statement otherwise. WDYT?

Reply:
True, done now

Reply:
We could also directly return `[CertifcateType.UNAVAILABLE]` and remove the initialization of the `UNAVAILABLE` variable and the mapping of it.

Reply:
Ah you're right, sorry for that, changed and cleaned up now